ldapauthenticator/ldapauthenticator.py

```diff
@@ -41,6 +41,33 @@ def _server_port_default(self):
         """
     )
 
+    server_ca_file = Unicode(
+        config=True,
+        help="""
+        Path of the CA certificate file for the *secure* LDAP server
+
+        If you're receiving `CERTIFICATE_VERIFY_FAILED` errors, you have to use this setting.
+        """
+    )
+
+    client_certificate_file = Unicode(
+        config=True,
+        help="""
+        Path of the certificate file for the LDAP client
+
+        If you're receiving `SSLV3_ALERT_HANDSHAKE_FAILURE` errors, you have to use this setting.
+        """
+    )
+
+    client_key_file = Unicode(
+        config=True,
+        help="""
+        Path of the key file for the LDAP client
+
+        If you're receiving `SSLV3_ALERT_HANDSHAKE_FAILURE` errors, you have to use this setting.
+        """
+    )
+
```

Comment:
I'm still not satisfied with the documentation that i wrote, but i think this could be left for another moment.